scripts/setup.sh

```diff
@@ -3,17 +3,11 @@
 cd ${OKTA_HOME}/${REPO}
 
 setup_service grunt
```

Comment:
nit:
I think we could remove this as well

Reply:
Yeah - we shouldn't need this anymore either. @manueltanzi-okta can you verify this?

Reply:
Will do 👍

Reply:
Verified that we still needed, will keep it